jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -1294,6 +1294,7 @@ public boolean onContentAdded()
                 case READY:
                     break;
 
+                case POSSIBLE:
```

Comment:
Can you explain how content can be added in POSSIBLE state?
I thought the idea was that we switched to POSSIBLE from REGISTERED state when the callback happened, but then we call handle and it is only withing handle that we then switch from POSSIBLE to PRODUCING state and then onContentAdded might be called.

Reply:
That shouldn't be possible.